nixos/modules/services/web-apps/isso.nix

```diff
@@ -0,0 +1,56 @@
+{ config, lib, pkgs, ... }:
+
+let
+  cfg = config.services.isso;
+
+  inherit (lib) mkEnableOption mkIf mkOption types;
+in {
+
+  options = {
+    services.isso = {
+      enable = mkEnableOption "Whether to enable isso, a commenting server similar to Disqus.";
+
+      user = mkOption {
+        type = types.str;
+        default = "isso";
+        description = "isso service user.";
+      };
+
+      group = mkOption {
+        type = types.str;
+        default = "isso";
+        description = "isso service group.";
+      };
+
+      config = mkOption {
```

Comment:
I would rather use `settings` so that Nix can be actually used to do the configuration rather than text (as per the relevant RFC for NixOS modules).

Reply:
Can you link that RFC? Actually I wanted to prevent migrating [all configuration params](https://posativ.org/isso/docs/configuration/server/) to NixOS.

Reply:
https://github.com/NixOS/rfcs/blob/master/rfcs/0042-config-option.md -- I meant rather to use INI format as a type / freeform rather than explicitly encoding *all parameters*.